### PR TITLE
Fix broken links

### PR DESCRIPTION
**Pull request by dveeden**
### What is changed, added or deleted? (Required)

Fix broken links

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

Related:
- https://github.com/pingcap/tidb-tools/pull/541
- https://github.com/pingcap/tidb-binlog/pull/1085

```
Link checker output file: link-checker/out.md
README.md
	ERROR	https://github.com/pingcap/docs/tree/release-5.3
		Not Found (HTTP error 404)
tidb-binlog/binlog-consumer-client.md
	ERROR	https://github.com/pingcap/tidb-tools/blob/master/tidb-binlog/proto/proto/binlog.proto
		Not Found (HTTP error 404)
tidb-binlog/tidb-binlog-overview.md
	ERROR	https://github.com/pingcap/tidb-binlog/blob/master/proto/binlog.proto
		Not Found (HTTP error 404)
adopters.md
	ERROR	http://company.funyours.co.jp/
		X509: certificate signed by unknown authority
	ERROR	https://nn.yjyz.com/
		Dial tcp4 47.113.27.42:443: connect: connection refused
connectors-and-apis.md
	ERROR	http://www.clifford.at/spl/spldoc/sql_mysql.html
		Lookup www.clifford.at on 168.63.129.16:53: read udp 172.17.0.2:45696->168.63.129.16:53: i/o timeout
```

### CODE REVIEW

File: connectors-and-apis.md [hunk 1]
```diff
@@ -74,7 +74,6 @@ The third-party APIs are not developed by Oracle. The following table lists the
 | Ruby           | MySQL/Ruby                               | `libmysqlclient`                 | Uses `libmysqlclient`. See [Section 27.12.1, “The MySQL/Ruby API”](https://dev.mysql.com/doc/refman/5.7/en/apis-ruby-mysqlruby.html) |
 |                | Ruby/MySQL                               | Native Driver                    | See [Section 27.12.2, “The Ruby/MySQL API”](https://dev.mysql.com/doc/refman/5.7/en/apis-ruby-rubymysql.html) |
 | Scheme         | `Myscsh`                                 | `libmysqlclient`                 | See [`Myscsh`](https://github.com/aehrisch/myscsh) |
-| SPL            | `sql_mysql`                              | `libmysqlclient`                 | See [`sql_mysql` for SPL](http://www.clifford.at/spl/spldoc/sql_mysql.html) |
```
Comment (TomShawn) [inline connectors-and-apis.md:77]: This document is not displayed on docs.pingcap.com and needs rigorous review and update. I don't think it's necessary to delete this entire row.

**Review by TomShawn — COMMENTED** [review]
Thanks for your fix!

The external link checker is **only for reference** and does not block merge. We do need to fix every link reported by the checker. It often reports false broken links merely due to poor connectivity. In fact, these links are through if you try it. In addition, some links are merely displayed in this GitHub repo, not on docs.pingcap.com or anywhere else.

File: adopters.md [hunk 2]
```diff
@@ -26,7 +26,6 @@ This is a list of TiDB adopters in various industries.
 |[Meituan](https://www.crunchbase.com/organization/meituan)|E-commerce|English [#1](https://pingcap.com/case-studies/migrating-from-mysql-to-a-scale-out-database-to-serve-our-290-million-monthly-users), [#2](https://pingcap.com/case-studies/how-we-use-a-mysql-alternative-to-avoid-sharding-and-provide-strong-consistency); [Chinese](https://pingcap.com/zh/case/user-case-meituandianping)|
 |[Happigo.com](https://www.crunchbase.com/organization/happigo-com)|E-commerce||
 |[Yimutian](https://www.crunchbase.com/organization/yimutian)|E-commerce||
-|[Youju Tech](https://nn.yjyz.com/)|E-commerce||
```
[the remark this answers is not in the document]
Comment (CaitinChen) [reply adopters.md:29]: @TomShawn Yes, please. Thanks.

File: adopters.md [hunk 3]
```diff
@@ -62,7 +61,7 @@ This is a list of TiDB adopters in various industries.
 |[Kunlun](https://www.crunchbase.com/organization/kunlun)|Gaming|[English](https://pingcap.com/case-studies/empowering-your-gaming-application-with-a-scale-out-newsql-database)|
 |[YOOZOO Games](https://www.crunchbase.com/organization/yoozoo-games)|Gaming||
 |[Seasun Games](https://www.crunchbase.com/organization/seasun)|Gaming||
-|[FUNYOURS JAPAN](http://company.funyours.co.jp/)|Gaming||
+|[FUNYOURS JAPAN](http://www.funyours.co.jp/#/company)|Gaming||
```
[the remark this answers is not in the document]
Comment (CaitinChen) [reply adopters.md:64]: So am I.

[the remark this answers is not in the document]
Comment (TomShawn) [reply adopters.md:64]:
```suggestion
|[FUNYOURS JAPAN](https://www.funyours.co.jp/#/top/pic1)|Gaming||
```

[the remark this answers is not in the document]
Comment (dveeden) [reply adopters.md:64]:
```suggestion
|[FUNYOURS JAPAN](http://company.funyours.co.jp/)|Gaming||
```

`http://company.funyours.co.jp/` redirects to `https://www.funyours.co.jp`, which doesn't seem to have a complete/correct chain of certificates.

```
$ curl -vL http://company.funyours.co.jp/
*   Trying 210.152.118.91:80...
* Connected to company.funyours.co.jp (210.152.118.91) port 80 (#0)
> GET / HTTP/1.1
> Host: company.funyours.co.jp
> User-Agent: curl/7.79.1
> Accept: */*
> 
* Mark bundle as not supporting multiuse
< HTTP/1.1 301 Moved Permanently
< Server: nginx
< Date: Thu, 18 Nov 2021 07:16:33 GMT
< Content-Type: text/html
< Content-Length: 178
< Connection: keep-alive
< Location: https://www.funyours.co.jp
< 
* Ignoring the response-body
* Connection #0 to host company.funyours.co.jp left intact
* Issue another request to this URL: 'https://www.funyours.co.jp/'
*   Trying 47.245.16.139:443...
* Connected to www.funyours.co.jp (47.245.16.139) port 443 (#1)
* ALPN, offering h2
* ALPN, offering http/1.1
* successfully set certificate verify locations:
*  CAfile: /etc/pki/tls/certs/ca-bundle.crt
*  CApath: none
* TLSv1.3 (OUT), TLS handshake, Client hello (1):
* TLSv1.3 (IN), TLS handshake, Server hello (2):
* TLSv1.2 (IN), TLS handshake, Certificate (11):
* TLSv1.2 (OUT), TLS alert, unknown CA (560):
* SSL certificate problem: unable to get local issuer certificate
* Closing connection 1
curl: (60) SSL certificate problem: unable to get local issuer certificate
More details here: https://curl.se/docs/sslcerts.html

curl failed to verify the legitimacy of the server and therefore could not
establish a secure connection to it. To learn more about this situation and
how to fix it, please visit the web page mentioned above.
```

[the remark this answers is not in the document]
Comment (TomShawn) [reply adopters.md]: OK. Let's use http://www.funyours.co.jp/#/company

File: connectors-and-apis.md (hunk 1 above)
Comment (dveeden) [reply connectors-and-apis.md:77]: Only removing the link is the best option for now.

**Review by TomShawn — APPROVED** [review]
LGTM

File: connectors-and-apis.md (hunk 1 above)
Comment (dveeden) [reply connectors-and-apis.md:77]: I've opened https://github.com/pingcap/docs/issues/6848 to address the other issues with this document

**Pull request merged**